src/modules/Cache.js

```diff
@@ -1,13 +1,25 @@
 import { getCurrentUrl, normalizeUrl } from '../helpers.js';
 
 export class Cache {
-	constructor() {
+	constructor(options) {
 		this.pages = {};
 		this.last = null;
+
+		const defaults = {
+			resolvePath: path => path
+		}
+		this.options = {
+			...defaults,
+			...options
+		};
+	}
+
+	normalizeUrl(url) {
```

Comment:
Can we change the naming here? This step does two things: normale + resolve, it probably shouldn't be called just normalize. Or maybe the normalizeUrl helper itself should be renamed more clearly to indicate it's just stripping the hostname?

Reply:
I agree, will change it to `resolveUrl`. `normalizeUrl` does also remove the hash if there is one. I would keep it as it is now, so that we don't get too many changes at once.